build_shared.sh

```diff
@@ -65,11 +65,15 @@ for platform in $PLATFORMS; do \
                      >"$KERNEL_TMP"/build.log 2>&1;
 
                 echo "Copying new kernel image ..."
-                cp "$KERNEL_TMP/arch/arm64/boot/Image.gz-dtb" "$KERNEL_TOP/common-kernel/kernel-dtb-$device"
+                obj="$OUT_OBJ_DEST/kernel-dtb-$device"
+                [ "$build_directory" ] && obj="$OUT_OBJ_DEST/kernel"
+                cp "$KERNEL_TMP/arch/arm64/boot/Image.gz-dtb" "$obj"
                 if [ $DTBO = "true" ]; then
+                    obj=$OUT_OBJ_DEST/dtbo-${device}.img
+                    [ "$build_directory" ] && obj="$OUT_OBJ_DEST/dtbo.img"
                     # shellcheck disable=SC2046
                     # note: We want wordsplitting in this case.
-                    $MKDTIMG create "$KERNEL_TOP"/common-kernel/dtbo-${device}.img $(find "$KERNEL_TMP"/arch/arm64/boot/dts -name "*.dtbo")
+                    $MKDTIMG create "$obj" $(find "$KERNEL_TMP"/arch/arm64/boot/dts -name "*.dtbo")
```

Comment:
Since above line should be retained, make a simple `cp` here, if `$obj` is defined.